types/zipkin-context-cls/zipkin-context-cls-tests.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    Tracer,
+    BatchRecorder,
+    jsonEncoder,
+} from 'zipkin';
+
+import CLSContext = require('zipkin-context-cls');
+import { HttpLogger } from 'zipkin-transport-http';
+
+// Setup the tracer to use http and implicit trace context
+const tracer = new Tracer({
+    ctxImpl: new CLSContext('zipkin'),
+    recorder: new BatchRecorder({
```

Comment:
This part in `recorder:` doesn't look relevant to testing `zipkin-context-cls` -- all we do to test that is construct it and pass it as `ctxImpl`.

Reply:
done.